src/components/organisms/OrganismArticle.vue

```diff
@@ -0,0 +1,73 @@
+<template>
+  <div class="mdc-card">
+    <div class="header">
+      <h1 class="mdc-typography--headline5 no-margin">{{ storage }}</h1>
+    </div>
+    <ul class="mdc-list">
+      <MoleculeArticleCard
+        class="mdc-list-item mdc-list-item--with-leading-image mdc-list-item--with-two-lines mdc-list-item--with-traili ng-icon"
+        v-for="i in 10"
+        :src="src"
+        :name="name"
+        :amount="amount"
+        :expiringIn="expiringIn"
+        :mhd="mhd"
+      ></MoleculeArticleCard>
+    </ul>
+  </div>
+</template>
+
+<script lang="ts" setup>
+
+import MoleculeArticleCard from '../molecules/MoleculeArticleCard.vue';
+
+defineProps({
+
+  name: {
+    type: String,
+    required: true,
+  },
+  src: {
+    type: String,
+    required: true,
+  },
+  mhd: {
+    type: String,
+    required: true,
+  },
+  expiringIn: {
+    type: String,
+    required: true,
+  },
+  amount: {
+    type: String,
+    required: true,
+  },
+  storage: {
+    type: String,
+    required: true,
+  },
+
+}
+);
```

Comment:
Code-Styling: remove line breaks
```suggestion
});
```